sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -95,14 +99,16 @@ private boolean springBootVersionMatches() {
         for (String acceptedVersion : acceptedVersions) {
             try {
                 if (this.matchSpringBootVersionFromManifest(acceptedVersion)) {
+                    LOGGER.debug("Current Spring Boot version matching succeeds in Spring Cloud Azure supported "
+                        + "versions [{}].", acceptedVersion);
```

Comment:
`The current Spring Boot version matches Spring Cloud Azure accepted version [{}]`